CHANGELOG.md

```diff
@@ -9,17 +9,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Support for multiline string literals.
 - Support for numeric literals prefixed by ampersands.
 - Support for identifiers prefixed by more than 2 ampersands.
 
+### Changed
+
+- `TextLiteralNode::getImage` now returns the text literal exactly as it appears in source code.
+
 ### Deprecated
 
+- `TextLiteralNode::getImageWithoutQuotes`, use `getValue` instead.
 - `DelphiTokenType.AMPERSAND`, as `&` is now lexed directly into numeric literals and identifiers.
 
 ### Fixed
 
 - Static char arrays weren't accepted for `'%s'` in `FormatArgumentType`.
 - Imaginary tokens could be erroneously matched by the lexer in valid source code.
+- `TextLiteralNode::getValue` threw a `NumberFormatException` on character escapes containing:
+  - Binary literals
+  - Digit separators
+- `TextLiteralNode::getValue` evaluated caret-notation character escapes incorrectly.
```

Comment:
`TextLiteralNode::getValue` didn't exist prior to this PR. These dot points should be removed from "Fixed" and summarised in an "Added" dot point, e.g.

```md
### Added

- **API:** `TextLiteralNode::getValue` method, which returns the effective contents of a string literal, including full support for character escapes.
```